src/webapp/components/dataset-table/DataSetTableConfig.tsx

```diff
@@ -124,20 +133,23 @@ export type CommonActionsProps<T> = {
     isActive: (data: T[]) => boolean;
     onAction: (action: TableAction) => void;
     navigateTo: ReturnType<typeof useNavigateTo>;
+    user: User;
 };
 
 export function getCommonActions<T extends ReferenceObject>(
```

Comment:
This is going to be used by a restricted set of classes, right (A| B). Maybe can be explicit with that, instead of ReferenceObject.